tests/cover/test_numerics.py

```diff
@@ -55,4 +59,13 @@ def test_decimals_can_disallow_nan(x):
 
 @given(decimals(allow_infinity=False))
 def test_decimals_can_disallow_inf(x):
+    assume(not x.is_snan())
     assert not math.isinf(x)
+
+
+@pytest.mark.parametrize('places', range(10))
+def test_decimals_have_correct_places(places):
+    @given(decimals(0, 10, allow_nan=False, places=places))
```

Comment:
Yep - you'd get a strategy over any finite decimal.  In practice, that's probably confined to the 64bit integer range due to the implementation of `integers()` drawing eight bytes.